### ISSUE TITLE
Write about new topics in ArXiV on 2022-03-01

### ISSUE THREAD

**Issue by wanted2** [wanted2/caineng.in#471]
**Is your feature request related to a problem? Please describe.**
*Write about new topics in ArXiV on 2022-03-01*
**Describe the solution you'd like**
* Topic 0: 
  * _Efficient Single Image Super-Resolution Using Dual Path Connections with  Multiple Scale Learning_
  * _Learning Explicit and Implicit Latent Common Spaces for Audio-Visual  Cross-Modal Retrieval_
  * _Learning Deep Semantic Model for Code Search using CodeSearchNet Corpus_
  * _A Novel Framework for Brain Tumor Detection Based on Convolutional  Variational Generative Models_
  * _Learning Multi-Object Dynamics with Compositional Neural Radiance Fields_
  * _A Novel Anomaly Detection Method for Multimodal WSN Data Flow via a  Dynamic Graph Neural Network_
  * _Asymmetric Hash Code Learning for Remote Sensing Image Retrieval_
  * _Deep learning based domain adaptation for mitochondria segmentation on  EM volumes_
  * _BLINC: Lightweight Bimodal Learning for Low-Complexity VVC Intra Coding_
  * _Bridging the Gap Between Patient-specific and Patient-independent  Seizure Prediction via Knowledge Distillation_
* Topic 1: 
  * _Reproducibility in Optimization: Theoretical Framework and Limits_
  * _Minimax in Geodesic Metric Spaces: Sion's Theorem and Algorithms_
  * _Long-Time Convergence and Propagation of Chaos for Nonlinear MCMC_
  * _Differentially Private Regression with Unbounded Covariates_
  * _Single Trajectory Nonparametric Learning of Nonlinear Dynamics_
  * _A PDE-Based Analysis of the Symmetric Two-Armed Bernoulli Bandit_
  * _Finding Safe Zones of policies Markov Decision Processes_
  * _Reachability analysis in stochastic directed graphs by reinforcement  learning_
  * _Generalized Bayesian Additive Regression Trees Models: Beyond  Conditional Conjugacy_
* Topic 2: 
  * _Graph Spring Network and Informative Anchor Selection for Session-based  Recommendation_
  * _Learning Multi-granularity User Intent Unit for Session-based  Recommendation_
  * _Hierarchical User Intent Graph Network forMultimedia Recommendation_
  * _Intention Adaptive Graph Neural Network for Category-aware Session-based  Recommendation_
  * _Multiple Choice Questions based Multi-Interest Policy Learning for  Conversational Recommendation_
  * _UA-FedRec: Untargeted Attack on Federated News Recommendation_
  * _Disentangled Graph Neural Networks for Session-based Recommendation_
  * _GIMIRec: Global Interaction Information Aware Multi-Interest Framework  for Sequential Recommendation_
  * _Multi-view Intent Disentangle Graph Networks for Bundle Recommendation_
* Topic 3: 
  * _Soundify: Matching Sound Effects to Video_
  * _Improving Content-Aware Video Streaming in Congested Networks with  In-Network Computing_
  * _DCNGAN: A Deformable Convolutional-Based GAN with QP Adaptation for  Perceptual Quality Enhancement of Compressed Video_
  * _Punctuation Restoration_
  * _Banding vs. Quality: Perceptual Impact and Objective Assessment_
  * _Nebula: Reliable Low-latency Video Transmission for Mobile Cloud Gaming_
  * _GenéLive! Generating Rhythm Actions in Love Live!_
  * _MMSys'22 Grand Challenge on AI-based Video Production for Soccer_
  * _Towards Using Clothes Style Transfer for Scenario-aware Person Video  Generation_
* Topic 4: 
  * _A QoE Model in Point Cloud Video Streaming_
  * _(2.5+1)D Spatio-Temporal Scene Graphs for Video Question Answering_
  * _repro_eval: A Python Interface to Reproducibility Measures of  System-oriented IR Experiments_
  * _Probably Reasonable Search in eDiscovery_
  * _Grep-BiasIR: A Dataset for Investigating Gender Representation-Bias in  Information Retrieval Results_
  * _Boosted Dense Retriever_
  * _Establishing Strong Baselines for TripClick Health Retrieval_
  * _Design Challenges for a Multi-Perspective Search Engine_
  * _Targeted Trojan-Horse Attacks on Language-based Image Retrieval_
* Topic 5: 
  * _Cross-modal Music Emotion Recognition Using Composite Loss-based  Embeddings_
  * _Video Background Music Generation with Controllable Music Transformer_
  * _Unsupervised Learning of Deep Features for Music Segmentation_
  * _Enhancing Affective Representations of Music-Induced EEG through  Multimodal Supervision and latent Domain Adaptation_
  * _S3T: Self-Supervised Pre-training with Swin Transformer for Music  Classification_
  * _A Novel Multi-Task Learning Method for Symbolic Music Emotion  Recognition_
  * _Learning long-term music representations via hierarchical contextual  constraints_
  * _Predicting emotion from music videos: exploring the relative  contribution of visual and auditory information to affective responses_
* Topic 6: 
  * _Event-centric Query Suggestion for Online News_
  * _Query Suggestion for Click-Absent Queries in Enterprise Search_
  * _Generating Clarifying Questions for Query Refinement in Source Code  Search_
  * _Trustworthy Anomaly Detection: A Survey_
* Topic 7: 
  * _Point cloud completion on structured feature map with feedback network_
  * _PCSCNet: Fast 3D Semantic Segmentation of LiDAR Point Cloud for  Autonomous Car using Point Convolution and Sparse Convolution Network_
  * _PMP-Net++: Point Cloud Completion by Transformer-Enhanced Multi-step  Point Moving Paths_
  * _Snowflake Point Deconvolution for Point Cloud Completion and Generation  with Skip-Transformer_
  * _Diversity in the Music Listening Experience: Insights from Focus Group  Interviews_
  * _Compressed Geometric Arrays for Point Cloud Processing_
  * _Patch-Based Deep Autoencoder for Point Cloud Geometry Compression_
  * _Reliable Inlier Evaluation for Unsupervised Point Cloud Registration_
  * _PointPCA: Point Cloud Objective Quality Assessment Using PCA-Based  Descriptors_
  * _Active Learning for Point Cloud Semantic Segmentation via  Spatial-Structural Diversity Reasoning_
* Topic 8: 
  * _Adversarial Example Devastation and Detection on Speech Recognition  System by Adding Random Noise_
  * _Exploring Adversarially Robust Training for Unsupervised Domain  Adaptation_
  * _SEC4SR: A Security Analysis Platform for Speaker Recognition_
  * _ARIA: Adversarially Robust Image Attribution for Content Provenance_
  * _Towards Robust Neural Image Compression: Adversarial Attack and Model  Finetuning_
  * _Mitigating Closed-model Adversarial Examples with Bayesian Neural  Modeling for Enhanced End-to-End Speech Recognition_
  * _Generative Adversarial Network-Driven Detection of Adversarial Tasks in  Mobile Crowdsensing_
  * _Real-time Over-the-air Adversarial Perturbations for Digital  Communications using Deep Neural Networks_
  * _Semi-Implicit Hybrid Gradient Methods with Application to Adversarial  Robustness_
* Topic 9: 
  * _The Gene of Scientific Success_
  * _Correcting Confounding via Random Selection of Background Variables_
  * _Relational Causal Models with Cycles:Representation and Reasoning_
  * _Diffusion Causal Models for Counterfactual Estimation_
  * _Deep Causal Reasoning for Recommendations_
  * _Learning Latent Causal Dynamics_
  * _Benign-Overfitting in Conditional Average Treatment Effect Prediction  with Linear Regression_
  * _Effect Identification in Cluster Causal Diagrams_
  * _Joint Geometry and Color Projection-based Point Cloud Quality Metric_
  * _Stochastic Causal Programming for Bounding Treatment Effects_
* Topic 10: 
  * _Neural Tangent Kernel Analysis of Deep Narrow Neural Networks_
  * _Approximation error of single hidden layer neural networks with fixed  weights_
  * _Relational Algebra and Calculus with SQL Null Values_
  * _Graph-Relational Domain Adaptation_
  * _Cost-effective Framework for Gradual Domain Adaptation with  Multifidelity_
  * _GPL: Generative Pseudo Labeling for Unsupervised Domain Adaptation of  Dense Retrieval_
  * _Conditional Extreme Value Theory for Open Set Video Domain Adaptation_
  * _Multi-Source Unsupervised Domain Adaptation via Pseudo Target Domain_
  * _Transferrable Contrastive Learning for Visual Domain Adaptation_
  * _Domain-Augmented Domain Adaptation_
* Topic 11: 
  * _Confidence Calibration for Object Detection and Segmentation_
  * _Musical Audio Similarity with Self-supervised Convolutional Neural  Networks_
  * _Local Sliced-Wasserstein Feature Sets for Illumination-invariant Face  Recognition_
  * _Guide Local Feature Matching by Overlap Estimation_
  * _R-Theta Local Neighborhood Pattern for Unconstrained Facial Image  Recognition and Retrieval_
  * _Local Directional Gradient Pattern: A Local Descriptor for Face  Recognition_
  * _Local Quadruple Pattern: A Novel Descriptor for Facial Image Recognition  and Retrieval_
  * _Local Gradient Hexa Pattern: A Descriptor for Face Recognition and  Retrieval_
* Topic 12: 
  * _LSB Based Non Blind Predictive Edge Adaptive Image Steganography_
  * _A Smoothing and Thresholding Image Segmentation Framework with Weighted  Anisotropic-Isotropic Total Variation_
  * _The Winning Solution to the iFLYTEK Challenge 2021 Cultivated Land  Extraction from High-Resolution Remote Sensing Image_
* Topic 13: 
  * _Synthesizing Photorealistic Images with Deep Generative Learning_
  * _Gapoera: Application Programming Interface for AI Environment of  Indonesian Board Game_
  * _CAISE: Conversational Agent for Image Search and Editing_
* Topic 14: 
  * _Biased or Not?: The Story of Two Search Engines_
  * _Tree-based Search Graph for Approximate Nearest Neighbor Search_
  * _Full-privacy secured search engine empowered by efficient genome-mapping  algorithms_
  * _Multimedia Datasets for Anomaly Detection: A Survey_
  * _Lightweight Multi-Drone Detection and 3D-Localization via YOLO_
  * _Amodal Panoptic Segmentation_
  * _AF$_2$: Adaptive Focus Framework for Aerial Imagery Segmentation_
  * _Perceptual Consistency in Video Segmentation_
* Topic 15: 
  * _3DRM:Pair-wise relation module for 3D object detection_
  * _Guidelines and evaluation for clinical explainable AI on medical image  analysis_
  * _Multi-Cue Adaptive Emotion Recognition Network_
* Topic 16: 
  * _A sampling-based approach for efficient clustering in large datasets_
  * _Use Image Clustering to Facilitate Technology Assisted Review_
  * _Optimal Clustering with Bandit Feedback_
  * _Human Motion Detection Using Sharpened Dimensionality Reduction and  Clustering_
* Topic 17: 
  * _Physics-Informed Graph Learning: A Survey_
  * _Unsupervised Graph Poisoning Attack via Contrastive Loss  Back-propagation_
* Topic 18: 
  * _Improving Fake News Detection by Using an Entity-enhanced Framework to  Fuse Diverse Multimodal Clues_
  * _Disentangling Hate in Online Memes_
  * _Detecting Harmful Memes and Their Targets_
  * _Using the left Gram matrix to cluster high dimensional data_
  * _Web-Based File Clustering and Indexing for Mindoro State University_
  * _Clustering by the Probability Distributions from Extreme Value Theory_
  * _Self-Evolutionary Clustering_
* Topic 19: 
  * _Cyclical Focal Loss_
  * _Nonconvex Extension of Generalized Huber Loss for Robust Learning and  Pseudo-Mode Statistics_
* Topic 20: 
  * _Anomalib: A Deep Learning Library for Anomaly Detection_
  * _Augment with Care: Contrastive Learning for the Boolean Satisfiability  Problem_
  * _STEREO: Scientific Text Reuse in Open Access Publications_
* Topic 21: 
  * _Clustering by Hill-Climbing: Consistency Results_
* Topic 22: 
  * _A hybrid 2-stage vision transformer for AI-assisted 5 class pathologic  diagnosis of gastric endoscopic biopsies_
  * _Robust Multi-Objective Bayesian Optimization Under Input Noise_
  * _A Robust Multi-Objective Bayesian Optimization Framework Considering  Input Uncertainty_
* Topic 23: 
* Topic 24: 
  * _FNR: A Similarity and Transformer-Based Approach to Detect Multi-Modal  Fake News in Social Media_
  * _Benchmarking Generative Latent Variable Models for Speech_
  * _AISHELL-NER: Named Entity Recognition from Chinese Speech_
  * _E-ffective: A Visual Analytic System for Exploring the Emotion and  Effectiveness of Inspirational Speeches_
  * _Speaker Identity Preservation in Dysarthric Speech Reconstruction by  Adversarial Speaker Adaptation_
  * _Speaker Adaptation Using Spectro-Temporal Deep Features for Dysarthric  and Elderly Speech Recognition_
  * _VCVTS: Multi-speaker Video-to-Speech synthesis via cross-modal knowledge  transfer from voice conversion_
* Topic 25: 
  * _General Cyclical Training of Neural Networks_
  * _Dynamic Object Comprehension: A Framework For Evaluating Artificial  Visual Perception_
* Topic 26: 
  * _Linear Model with Local Differential Privacy_
  * _Differentially Private Estimation of Heterogeneous Causal Effects_
  * _Variance reduced stochastic optimization over directed graphs with row  and column stochastic weights_
  * _Differential privacy for symmetric log-concave mechanisms_
  * _A Survey on Patients Privacy Protection with Stganography and Visual  Encryption_
* Topic 27: 
* Topic 28: 
  * _A Survey of Ad Hoc Teamwork: Definitions, Methods, and Open Problems_
  * _DL-SLOT: Dynamic Lidar SLAM and Object Tracking Based On Graph  Optimization_
  * _Bidding Agent Design in the LinkedIn Ad Marketplace_
* Topic 29: 
  * _Provable Stochastic Optimization for Global Contrastive Learning: Small  Batch Does Not Harm Performance_
* Topic 30: 
* Topic 31: 
  * _Efficient and Differentiable Conformal Prediction with General Function  Classes_
  * _Click-Through Rate Prediction in Online Advertising: A Literature Review_
  * _CSCNet: Contextual Semantic Consistency Network for Trajectory  Prediction in Crowded Spaces_
  * _M2I: From Factored Marginal Trajectory Prediction to Interactive  Prediction_
  * _Selective Credit Assignment_
* Topic 32: 
* Topic 33: 
* Topic 34: 
  * _MultiRes-NetVLAD: Augmenting Place Recognition Training with  Low-Resolution Imagery_
* Topic 35: 
* Topic 36: 
  * _Deconstructing Distributions: A Pointwise Framework of Learning_
  * _Large-scale Personalized Video Game Recommendation via Social-aware  Contextualized Graph Neural Network_
  * _How player and opponent personalities influence cooperative gameplay_
  * _Player Dominance Adjustment in Games_
  * _Fighting Game Commentator with Pitch and Loudness Adjustment Utilizing  Highlight Cues_
  * _Integrating psychotherapy practices and gamified elements in novel game  mechanics for stress relief_
  * _A Complete Criterion for Value of Information in Soluble Influence  Diagrams_
* Topic 37: 
* Topic 38: 
  * _Why Do We Click: Visual Impression-aware News Recommendation_
  * _Assisting News Media Editors with Cohesive Visual Storylines_
* Topic 39: 
  * _FUNQUE: Fusion of Unified Quality Evaluators_
* Topic 40: 
  * _6D Rotation Representation For Unconstrained Head Pose Estimation_
* Topic 41: 
  * _Temporal Subtyping of Alzheimer's Disease Using Medical Conditions  Preceding Alzheimer's Disease Onset in Electronic Health Records_
  * _Subtyping brain diseases from imaging data_
  * _Graph Convolutional Networks for Multi-modality Medical Imaging:  Methods, Architectures, and Clinical Applications_
* Topic 42: 
  * _Learning to Identify Perceptual Bugs in 3D Video Games_
  * _On PAC-Bayesian reconstruction guarantees for VAEs_
* Topic 43: 
  * _Geometry of the Minimum Volume Confidence Sets_
* Topic 44: 
* Topic 45: 
  * _When do GANs replicate? On the choice of dataset size_
  * _MAVE: A Product Dataset for Multi-source Attribute Value Extraction_
* Topic 46: 
  * _Extraction of Product Specifications from the Web -- Going Beyond Tables  and Lists_
* Topic 47: 
  * _Multiscale Crowd Counting and Localization By Multitask Point  Supervision_
  * _Sobolev Transport: A Scalable Metric for Probability Measures with Graph  Metrics_
* Topic 48: 
  * _The Quarks of Attention_
* Topic 49: 
  * _A fair pricing model via adversarial learning_
  * _Implementing simple spectral denoising for environmental audio  recordings_

**Describe alternatives you've considered**
None
**Additional context**
None

**Comment by wanted2** [comment]
This issue was created by an aggregate batch
